java.asciidoc

```diff
@@ -0,0 +1,298 @@
+= Pony for Java
```

Comment:
yeah so the previous 

```
=
==
===
```

was correct.

Sorry, its been a long two weeks and I'm pretty tired.

Going back to what was previously there.